test-cypress/integration/sparql-editor/sparql-editor.spec.js

```diff
@@ -52,4 +52,30 @@ describe('Sparql editor', () => {
         // And yasr tabs should not be visible
         YasrSteps.getResultHeader().should('not.be.visible');
     });
+
+    it('Should paste and add prefixes', () => {
+        // Given I have opened the workbench
+        // When I visit the sparql editor page
+        SparqlEditorSteps.visitSparqlEditorPage();
+        YasqeSteps.clearEditor();
+        // I paste a query into the editor
+        cy.pasteIntoCodeMirror('.CodeMirror', 'select distinct ?x ?Person  where {\n' +
+            '?x a afn:Person .\n' +
+            '?x afn:preferredLabel ?Person .\n' +
+            '?doc afn:containsMention / pub-old:hasInstance ?x .\n' +
+            '}');
+        // Wait, so that Yasqe can have time to replace the existing query with the new one
+        cy.wait(100);
```

Comment:
I'm worried about this explicit wait. Can you try replacing it with a conditional wait. See the `verifyQueryAreaContains` in the sparql-commands.js for example